### PR TITLE
fix edge device state sync error

### PR DESCRIPTION
**Pull request by JiaweiGithub**
Signed-off-by: jiawei <jiawei.liu@daocloud.io>

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/devicetwin/dttype/types.go [hunk 1]
```diff
@@ -23,6 +23,17 @@ type Device struct {
...
+// DeviceCloudMsg used to synchronize device data to the cloud
+type DeviceCloudMsg struct {
+	ID             string              `json:"id,omitempty"`
+	Name           string              `json:"name,omitempty"`
+	Description    string              `json:"description,omitempty"`
+	State          string              `json:"state,omitempty"`
+	LastOnlineTime string              `json:"lastOnlineTime,omitempty"`
+	Attributes     map[string]*MsgAttr `json:"attributes,omitempty"`
+	Twin           map[string]*MsgTwin `json:"twin,omitempty"`
+}
```
Comment (fisherxu) [inline edge/pkg/devicetwin/dttype/types.go:35]: What's the difference with `Device`?

**Review by fisherxu — COMMENTED** [review]
/approve

File: edge/pkg/devicetwin/dttype/types.go (hunk 1 above)
Comment (JiaweiGithub) [reply edge/pkg/devicetwin/dttype/types.go:35]: The original device has a field last_online, which is related to the database, but this name is not suitable for use on the cloud. The database fields are not suitable for adjustment, so I think this is the smallest change plan.

**Comment by Shelley-BaoYue** [comment]
cc @WillardHu

**Review by Shelley-BaoYue — COMMENTED** [review]
Please cherry-pick to release-1.18

**Pull request merged**